docs/content/about-iota/FAQ.mdx

```diff
@@ -99,22 +99,121 @@ For current token holders:
+   - [How to use Balance Finder](https://docs.iota.org/about-iota/iota-wallet/getting-started)
+
+3. After finding your addresses, go to the [IOTA Wallet Dashboard](https://wallet.iota.org) and check:
+   - **Migration Tab**: For unmigrated Stardust balances  
+     [Migrate Stardust Assets](https://docs.iota.org/about-iota/iota-wallet/migrate-assets)
+   - **Vesting Tab**: For airdropped or locked funds from the October 2024 upgrade  
+     [Manage Vesting and Time-Locked Assets](https://docs.iota.org/about-iota/iota-wallet/vesting)
+
+## 15. I'm Sure My Funds Are in a Specific Address But They’re Not Showing
+
+1. Connect that address to the [Wallet Dashboard](https://wallet.iota.org).
+2. Navigate to the **Migration** tab and copy your legacy address.
+3. Paste it into the [Archived Tangle Explorer](https://explorer.iota.org/mainnet) to check past balances.
+
+If you don’t see your funds there, they are likely in a **different address**. Run the **Balance Finder** again.
+
+## 16. I Can't Connect My New Accounts to the Wallet Dashboard
+
+This is a known issue. Please follow these steps:
+
+1. Go to `IOTA Wallet > Apps Tabs > Active Connections > Wallet Dashboard`.
+2. **Disconnect all** accounts.
+3. Reconnect them again from the Wallet Dashboard.
+
+## 17. I Lost My Mnemonic. Can I Still Migrate?
+
+Yes—if you still have your **Stronghold file** and its **password**, you can extract the seed:
+
+1. Use **Firefly v2.1.0+** to extract the seed.
+   - [Restore From Seed or Stronghold](https://docs.iota.org/about-iota/iota-wallet/how-to/restore-account)
+2. Import the extracted mnemonic into the new wallet:
+   - [Getting Started with the IOTA Wallet](https://docs.iota.org/about-iota/iota-wallet/getting-started)
+
+:::note
+Restoring from seed is not an industry standard and may only work with IOTA Foundation-supported wallets. Once restored, transfer your funds to a wallet with a **known mnemonic**.
+:::
+
+## 18. Importing a Stronghold Crashes Firefly
+
+If you still have a Firefly profile:
+- Follow this guide to export the seed:  
+  [Back up a Seed from a Firefly Profile](https://docs.iota.org/about-iota/iota-wallet/how-to/restore-account#back-up-a-seed-from-a-firefly-profile)
+
+If you **don’t** have the profile:
+1. Download [Firefly Shimmer](https://firefly.iota.org/).
+2. Create a Shimmer profile using your **mnemonic**.
+3. Log out and export the seed:  
+   [Guide to Exporting Seed](https://docs.iota.org/about-iota/iota-wallet/how-to/restore-account#back-up-a-seed-from-a-firefly-profile)
+
+## 19. What Does It Mean If My Account is Marked as "Legacy"?
+
+This means your addresses originate from the **"Chrysalis rotating address era"**. These addresses are considered outdated and may not be recognized by newer wallets.
+
+:::info
+**We recommend** moving your funds out of legacy addresses to ensure compatibility and accessibility.
+:::
+
+## 20. Will there be any migration steps for users?
```

Comment:
This is not really true and we have to update it, users might need do migrate and we should point them to the tutorial about it

Reply:
I added a link pointing them to the migrate Stardust assets page

Reply:
@Ginowine I dont see that, the whole point is still false 
> No manual migration is needed:

this is certainly not true, users might need migration and we have to explain them that

Reply:
I understand what you wanted me to do and have implemented it. Thanks.